### ISSUE TITLE
Clean up deprecated packages

### ISSUE THREAD

**Issue by cobraz** [bjerkio/website#135]
- [ ] urix@npm:0.1.0 is deprecated: Please see https://github.com/lydell/urix#deprecated
- [ ] resolve-url@npm:0.2.1 is deprecated: https://github.com/lydell/resolve-url#deprecated
- [ ] core-js@npm:2.6.12 is deprecated: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
- [ ] cheerio-select-tmp@npm:0.1.1 is deprecated: Use cheerio-select instead
- [ ] @hapi/joi@npm:15.1.1 is deprecated: Switch to 'npm install joi'
- [ ] fsevents@npm:2.3.2: Implicit dependencies on node-gyp are discouraged
- [ ] @hapi/bourne@npm:1.3.2 is deprecated: This version has been deprecated and is no longer supported or maintained
- [ ] @hapi/address@npm:2.1.4 is deprecated: Moved to 'npm install @sideway/address'
- [ ] @hapi/topo@npm:3.1.6 is deprecated: This version has been deprecated and is no longer supported or maintained
- [ ] @hapi/hoek@npm:8.5.1 is deprecated: This version has been deprecated and is no longer supported or maintained
- [ ] @graphql-toolkit/common@npm:0.10.7 is deprecated: GraphQL Toolkit is deprecated and merged into GraphQL Tools, so it will no longer get updates. Use GraphQL Tools instead to stay up-to-date! Check out https://www.graphql-tools.com/docs/migration-from-toolkit for migration and https://the-guild.dev/blog/graphql-tools-v6 for new changes.
- [ ] gatsby-image@npm:2.11.0 is deprecated: This package has been deprecated in favor of gatsby-plugin-image. The migration guide can be found here: https://www.gatsbyjs.com/docs/reference/release-notes/image-migration-guide/.
- [ ] request@npm:2.88.2 is deprecated: request has been deprecated, see https://github.com/request/request/issues/3142
- [ ] har-validator@npm:5.1.5 is deprecated: this library is no longer supported
- [ ] request-promise-native@npm:1.0.9 is deprecated: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
- [ ] postcss-cssnext@npm:3.1.0 is deprecated: 'postcss-cssnext' has been deprecated in favor of 'postcss-preset-env'. Read more at https://moox.io/blog/deprecating-cssnext/
- [ ] browserslist@npm:2.11.3 is deprecated: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
- [ ] extract-text-webpack-plugin@npm:3.0.2 is deprecated: Deprecated. Please use https://github.com/webpack-contrib/mini-css-extract-plugin
- [ ] popper.js@npm:1.16.1 is deprecated: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
- [ ] eslint-loader@npm:2.2.1 is deprecated: This loader has been deprecated. Please use eslint-webpack-plugin
- [ ] chokidar@npm:2.1.8 is deprecated: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
- [ ] fsevents@npm:1.2.13 is deprecated: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
- [ ] fsevents@npm:1.2.13 is deprecated: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
- [ ] evp_bytestokey@npm:1.0.3: Implicit dependencies on node-gyp are discouraged
- [ ] babel-eslint@npm:10.1.0 is deprecated: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
- [ ] nan@npm:2.14.2: Implicit dependencies on node-gyp are discouraged
- [ ] @endemolshinegroup/cosmiconfig-typescript-loader@npm:3.0.2 [f10bb] doesn't provide typescript (pb37b9), requested by ts-node
- [ ] @sanity/base@npm:2.7.2 [d5032] doesn't provide react-is (p901b4), requested by styled-components
- [ ] @sanity/base@npm:2.7.2 [d5632] doesn't provide react-is (pf0a53), requested by styled-components
- [ ] @sanity/base@npm:2.7.2 [f7153] doesn't provide react-is (pcd3ec), requested by styled-components
- [ ] @sanity/core@npm:2.7.2 doesn't provide @babel/core (pe8b7b), requested by @babel/plugin-proposal-class-properties
- [ ] @sanity/core@npm:2.7.2 doesn't provide @babel/core (p89793), requested by @babel/preset-env
- [ ] @sanity/core@npm:2.7.2 doesn't provide @babel/core (pa90b8), requested by @babel/preset-react
- [ ] @sanity/core@npm:2.7.2 doesn't provide @babel/core (pfc5c6), requested by @babel/preset-typescript
- [ ] @sanity/core@npm:2.7.2 doesn't provide @babel/core (pda9da), requested by @babel/register
- [ ] @sanity/core@npm:2.7.2 doesn't provide prop-types (pbeb66), requested by @sanity/server
- [ ] @sanity/core@npm:2.7.2 doesn't provide react (p12281), requested by @sanity/server
- [ ] @sanity/core@npm:2.7.2 doesn't provide react-dom (pf59a8), requested by @sanity/server
- [ ] @sanity/core@npm:2.7.2 doesn't provide webpack (pbdbcd), requested by @sanity/webpack-integration
- [ ] @sanity/default-layout@npm:2.7.2 [d5632] doesn't provide react-is (pe11a4), requested by styled-components
- [ ] @sanity/desk-tool@npm:2.7.2 [d5632] doesn't provide react-dom (pbc098), requested by @reach/auto-id
- [ ] @sanity/desk-tool@npm:2.7.2 [d5632] doesn't provide react-dom (pd4b86), requested by @sanity/ui
- [ ] @sanity/desk-tool@npm:2.7.2 [d5632] doesn't provide react-dom (pe62d6), requested by boundless-arrow-key-navigation
- [ ] @sanity/desk-tool@npm:2.7.2 [d5632] doesn't provide react-dom (p71001), requested by styled-components
- [ ] @sanity/desk-tool@npm:2.7.2 [d5632] doesn't provide react-dom (p221bd), requested by @sanity/form-builder
- [ ] @sanity/desk-tool@npm:2.7.2 [d5632] doesn't provide react-is (pf58db), requested by styled-components
- [ ] @sanity/field@npm:2.7.2 [a31ea] doesn't provide prop-types (pfe517), requested by @sanity/base
- [ ] @sanity/field@npm:2.7.2 [a31ea] doesn't provide react-dom (pe00ae), requested by @sanity/ui
- [ ] @sanity/field@npm:2.7.2 [a31ea] doesn't provide react-dom (p19d9d), requested by styled-components
- [ ] @sanity/field@npm:2.7.2 [a31ea] doesn't provide react-dom (p3da45), requested by @sanity/base
- [ ] @sanity/field@npm:2.7.2 [a31ea] doesn't provide react-is (pee1f6), requested by styled-components
- [ ] @sanity/form-builder@npm:2.7.2 [a31ea] doesn't provide react-is (p9d5d3), requested by styled-components
- [ ] @sanity/plugin-loader@npm:2.7.2 doesn't provide webpack (pc9d5b), requested by @sanity/webpack-integration
- [ ] @sanity/portable-text-editor@npm:0.1.25 [f7153] doesn't provide react-dom (p6604b), requested by styled-components
- [ ] @sanity/portable-text-editor@npm:0.1.25 [f7153] doesn't provide react-dom (p45765), requested by @sanity/slate-react
- [ ] @sanity/portable-text-editor@npm:0.1.25 [f7153] doesn't provide react-is (p88ac5), requested by styled-components
- [ ] @sanity/webpack-integration@npm:2.7.2 [032dc] doesn't provide caniuse-lite (p24040), requested by postcss-cssnext
- [ ] @sanity/webpack-integration@npm:2.7.2 [cef76] doesn't provide caniuse-lite (p5e547), requested by postcss-cssnext
- [ ] bjerkio-studio@workspace:studio provides react (p7044d) with version 16.14.0, which doesn't satisfy what @sanity/desk-tool and some of its descendants request
- [ ] bjerkio-web@workspace:web doesn't provide @reach/router (p89a98), requested by gatsby-link
- [ ] bjerkio-web@workspace:web doesn't provide @theme-ui/css (p210ca), requested by @theme-ui/theme-provider
- [ ] bjerkio-web@workspace:web provides gatsby (pf118e) with version 2.32.11, which doesn't satisfy what gatsby-source-filesystem requests
- [ ] bjerkio-web@workspace:web provides gatsby (pd0a4f) with version 2.32.11, which doesn't satisfy what gatsby-transformer-remark requests
- [ ] bjerkio-web@workspace:web doesn't provide graphql (pa7419), requested by @graphql-codegen/typescript
- [ ] bjerkio-web@workspace:web doesn't provide graphql (p59eaf), requested by @graphql-codegen/typescript-operations
- [ ] bjerkio-web@workspace:web doesn't provide graphql (p40151), requested by @graphql-codegen/cli
- [ ] bjerkio-web@workspace:web doesn't provide graphql (pd5ae8), requested by @graphql-codegen/fragment-matcher
- [ ] bjerkio-web@workspace:web doesn't provide graphql (p2de6d), requested by gatsby-plugin-graphql-codegen
- [ ] bjerkio-web@workspace:web provides react (p678d4) with version 17.0.2, which doesn't satisfy what @theme-ui/color-modes and some of its descendants request
- [ ] bjerkio-web@workspace:web provides react (pbbfbf) with version 17.0.2, which doesn't satisfy what @theme-ui/mdx requests
- [ ] bjerkio-web@workspace:web provides react (pf320d) with version 17.0.2, which doesn't satisfy what @theme-ui/components and some of its descendants request
- [ ] bjerkio-web@workspace:web provides react (pe522c) with version 17.0.2, which doesn't satisfy what @theme-ui/theme-provider requests
- [ ] bjerkio-web@workspace:web provides react (pd3dfe) with version 17.0.2, which doesn't satisfy what gatsby and some of its descendants request
- [ ] bjerkio-web@workspace:web provides react (pa9625) with version 17.0.2, which doesn't satisfy what react-burger-menu requests
- [ ] bjerkio-web@workspace:web provides react (p5c747) with version 17.0.2, which doesn't satisfy what theme-ui and some of its descendants request
- [ ] bjerkio-web@workspace:web provides react-dom (pae3ec) with version 17.0.2, which doesn't satisfy what gatsby and some of its descendants request
- [ ] bjerkio-web@workspace:web provides react-dom (pef495) with version 17.0.2, which doesn't satisfy what react-burger-menu requests
- [ ] bjerkio-web@workspace:web provides theme-ui (p2fbec) with version 0.4.0-rc.14, which doesn't satisfy what gatsby-plugin-theme-ui requests
- [ ] bjerkio-web@workspace:web doesn't provide typescript (p39727), requested by @graphql-codegen/cli
- [ ] bjerkio-web@workspace:web doesn't provide typescript (pe72e4), requested by gatsby-plugin-graphql-codegen
- [ ] gatsby-plugin-typescript@npm:2.12.1 doesn't provide gatsby (pcb7aa), requested by babel-plugin-remove-graphql-queries

We also need to upgrade packages. Probably after #130

**Comment by dkostenko-bjerk** [comment]
@cobraz What command did you use for this list?
